src/vaadin-dialog-draggable-mixin.html

```diff
@@ -0,0 +1,47 @@
+<script>
+  /**
+   * @namespace Vaadin
+   */
+  window.Vaadin = window.Vaadin || {};
+
+  /**
+   * @polymerMixin
+   * @memberof Vaadin
+   */
+  Vaadin.DialogDraggableMixin = subclass => class VaadinDialogDraggableMixin extends subclass {
+    ready() {
+      super.ready();
+      this._addDraggableEventListeners();
+    }
+
+    _addDraggableEventListeners() {
+      let originalBounds;
+      let originalMouse;
+      this._stopDrag = () => {
```

Comment:
Not sure why these methods are all defined here instead of as regular class members... If the problem is with `this` context, you can bind them in `ready()`: `this._stopDrag = this._stopDrag.bind(this)` etc

Reply:
It made sense to me when the mixins were part of the main file, I thought it was cleaner to have all the logic for the features encapsulated in the 2 methods. I have refactored both mixins now.